### DISCUSSION TITLE
FRR - replacement for Quagga

### DISCUSSION THREAD

**Discussion by Dukat-Gul** [Freetz-NG/freetz-ng#992] [Unerfüllte Wünsche]
As there has been a recent Announcement of [FRR release 10.0 ](https://github.com/FRRouting/frr/releases) would it be cool to replace / update Quagga with a more recent supported routing capability. Is this something that might be possible?

**Comment by fda77** [comment]
You could take a random directory from 'make/pkgs/', maybe 'make/pkgs/quagga', copy it to 'make/pkgs/frr/' and apply anything needed to make it work. Run `make help` to see usefull commands
